src/FunctionAppResolver.ts

```diff
@@ -20,6 +20,6 @@ export class FunctionAppResolver implements AppResourceResolver {
     }
 
     public matchesResource(resource: AppResource): boolean {
-        return resource.type.toLowerCase() === 'microsoft.web/sites' && !!resource.kind?.includes('functionapp');
+        return resource.type.toLowerCase() === 'microsoft.web/sites' && !!resource.kind?.includes('functionapp') && !resource.kind?.includes('workflowapp');
```

Comment:
A comment on that part would be good. Maybe also break up the expression into three lines for ease of reading.

Reply:
Will do. Also been thinking about if we should put the "is this an X app" util functions into azure utils.